src/events/components/EventImage.tsx

```diff
@@ -1,18 +1,17 @@
 import ResponsiveImage from 'common/components/ResponsiveImage';
 import IResponsiveImage, { IResponsiveImageSizes } from 'common/models/ResponsiveImage';
 import React, { FC, ImgHTMLAttributes } from 'react';
-import { ICompanyEvent } from '../models/Event';
 import { DefaultEventImage } from './DefaultEventImage';
 
 export interface IProps extends ImgHTMLAttributes<HTMLImageElement> {
-  image: IResponsiveImage | null;
-  companyEvents: ICompanyEvent[];
+  images: IResponsiveImage[];
   size: keyof IResponsiveImageSizes;
   color?: string;
 }
 
-const EventImage: FC<IProps> = ({ image, companyEvents: [companyEvent], color, ...props }) => {
-  const img = companyEvent ? companyEvent.company.image : image;
+const EventImage: FC<IProps> = ({ images, color, ...props }) => {
+  // TODO: implement a way to display multiple event images.
```

Comment:
Is there an issue for this? If not, could you make one? 📓

Reply:
We have our good old boi #45